packages/cli/package.json

```diff
@@ -195,7 +195,13 @@
       "@loopback/monorepo": "^0.5.10",
       "@loopback/tsdocs": "^4.0.10",
       "@loopback/example-references-many": "^6.0.10",
-      "@loopback/sequelize": "^0.2.0"
+      "@loopback/sequelize": "^0.2.0",
+      "loopback-connector": "^5.3.0",
```

Comment:
@yeshamavani We need to find a way to detect a loopback official connector and the update that specific dependency only.